states.go

```diff
@@ -13,7 +13,8 @@ const (
 	TaskState_TASK_LOST     TaskState = "TASK_LOST"
 	TaskState_TASK_ERROR    TaskState = "TASK_ERROR"
 	// Custom eremetic states
-	TaskState_TASK_QUEUED TaskState = "TASK_QUEUED"
+	TaskState_TASK_QUEUED      TaskState = "TASK_QUEUED"
+	TaskState_TASK_TERMINATING TaskState = "TASK_TERMINATING"
```

Comment:
When it's successfully killed it ends up in TASK_LOST
TASK_KILLED is also used by mesos as a final state, and I intended this state to more log the state change that the kill command was received, but it is not yet terminated